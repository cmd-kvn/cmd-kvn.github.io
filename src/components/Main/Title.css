p {
    font-size: 24px;
    font-family: Cabin
}

span {
    font-size: 72px;
    font-family: Rock Salt;
}