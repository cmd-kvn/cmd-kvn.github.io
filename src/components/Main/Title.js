import React from 'react';

import './Title.css';

const Title = () => {
    return (
        <div>
            <p>
                <span className='name'>Kevin Wong</span> <br />
                Full Stack Web Developer &nbsp; - &nbsp; React.js / Node.js
            </p>
            <hr />
        </div>
    );
};

export default Title;
