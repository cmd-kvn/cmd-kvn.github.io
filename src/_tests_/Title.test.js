// /* eslint-disable */
import React from 'react';
// import ReactDOM from 'react-dom';
// import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Title from '../components/Main/Title';

describe('Title', function() {
    it('runs the Title', function() {
        const rendered = shallow(<Title />);
        expect(toJson(rendered)).toMatchSnapshot();
    });
});

// TODO: investigate/disable these linting errors and
// determine which npm packages to actually use
