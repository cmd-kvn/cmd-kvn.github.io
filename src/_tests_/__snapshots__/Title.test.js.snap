exports[`Title runs the Title 1`] = `
<div>
  <p>
    <span
      className="name">
      Kevin Wong
    </span>
     
    <br />
    Full Stack Web Developer   -   React.js / Node.js
  </p>
  <hr />
</div>
`;
